### ISSUE TITLE
Upgrade to 1.20 seems not to be there?

### ISSUE THREAD

**Issue by develroo** [ian612/sfrpgbb#38]
I have a game (Stars without Number) which uses the sfrpgbb  module. When I launch it, it fails to load and a warning is given to upgrade sfrpgbb to 1.20

![Screenshot from 2022-12-19 00-00-26](https://user-images.githubusercontent.com/7777993/208326488-23619474-26e3-406f-b17b-7d6fd5c5f860.png)

However, there seem to be no updates via the modules page or indeed on this github. The last release from Sep 20 2022 is still 1.11

![Screenshot from 2022-12-19 00-04-03](https://user-images.githubusercontent.com/7777993/208326564-d497ac08-b428-4ad6-ae18-ad4010983eaf.png)

Can someone explain why it is prompting for an update when none are available?

Thanks in advance.

**Comment by ian612** [comment]
Hi, this was an issue with my defining of the manifest system.json on Foundry’s package manager. I accidentally set it to the working branch instead of the 1.1.1 release. When I started making some updates that file with the version number got overridden, but the installer was still pointed back to the release. I think I’ve remedied the issue, please let me know if it’s still broken.

The current stable version is 1.1.1, version 1.2 is not released yet.

**Comment by develroo** [comment]
Sadly, it does not seem to resolve the issue. I removed the module and reinstalled it. I even removed the World and created a new 'test' one, but the error is still there? I suspect the error is not with the module itself as I have scanned the files and there is no string "1.20" anywhere, but it might be the update mechanism Fvtt is using to check your manifest version, which might need to be updated on the website. 

Hope that helps
